<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWidget</class>
 <widget class="QDialog" name="ChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="sendPushButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>370</y>
     <width>91</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="chatTextEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>581</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputTextEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>491</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>41</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="showNameLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 255, 210);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="digitCount">
    <number>19</number>
   </property>
  </widget>
  <widget class="QPushButton" name="addFrdPushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="setPushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 255, 185);</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>231</width>
     <height>501</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>我的好友</string>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/new/prefix1/img/icon.png</normaloff>:/new/prefix1/img/icon.png</iconset>
    </property>
   </column>
  </widget>
  <zorder>label_3</zorder>
  <zorder>sendPushButton</zorder>
  <zorder>chatTextEdit</zorder>
  <zorder>inputTextEdit</zorder>
  <zorder>label</zorder>
  <zorder>showNameLabel</zorder>
  <zorder>lcdNumber</zorder>
  <zorder>addFrdPushButton</zorder>
  <zorder>setPushButton</zorder>
  <zorder>treeWidget</zorder>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
